app/views/movies/_movie_search.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= button_to 'Find Top Rated Movies', movies_path, params: { q: :top_rated }, method: :get %>
+
+<%= form_with url: movies_path, method: :get, local: true do |f| %>
+
+  <%= f.search_field :title, placeholder: 'Search by movie title' %>
```

Comment:
We should consider refactoring this so that both search terms are 'q' or 'query' instead of 'q' and 'title'; we could also accomplish this by putting the button inside the form